docs/wearables/providers/data-attributions.mdx

```diff
@@ -53,6 +53,44 @@ The following provider integrations would tag data with Source Type:
 <sup>[1]</sup> Only data from Apple Watch, iPhone and Apple Health app are tagged. Data from third-party apps
 (like the Oura iOS app) are tagged as `unknown` currently.
 
+## Device ID
+
+Device ID indicates the specific device that collected the data, when this information is available with certainty.
+
+For both summary and timeseries data types, it is available at `source.device_id`. When the originating device cannot be determined, this field is `null`.
+
+### Device Attribution Coverage
+
+Device attribution is available when Junction can confidently identify the originating device:
+
+| Provider        | Coverage | Notes |
+| --------------- | -------- | ----- |
+| Withings        | All payloads except Body, Body Fat and Body Weight | Regardless of the number of registered devices |
+| Polar           | Workout and Sleep payloads | Regardless of the number of registered devices |
+| All other providers and resources| All payloads | When exactly one device is registered for the user-provider connection |
```

Comment:
It is not "payloads" that we are attributing to devices but the resource being sent in the payload.

Reply:
also _reported_ is a better choice of word, than _registered_ which typically implies an explicit act of registration.